src/main/java/seedu/address/MainApp.java

```diff
@@ -69,25 +69,25 @@ public void init() throws Exception {
     }
 
     /**
-     * Returns a {@code ModelManager} with the data from {@code storage}'s address book and {@code userPrefs}. <br>
-     * The data from the sample address book will be used instead if {@code storage}'s address book is not found,
-     * or an empty address book will be used instead if errors occur when reading {@code storage}'s address book.
+     * Returns a {@code ModelManager} with the data from {@code storage}'s show list and {@code userPrefs}. <br>
+     * The data from the sample show liost will be used instead if {@code storage}'s show list is not found,
```

Comment:
There is a minor typo here. `show liost -> show list`

Reply:
yup